packages/backend/src/generated/routes.ts

```diff
@@ -434,19 +434,13 @@ const models: TsoaRoute.Models = {
         },
     },
     // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
-    'Partial_Pick_OrganizationMemberProfile.role__': {
-        dataType: 'refAlias',
-        type: {
-            dataType: 'nestedObjectLiteral',
-            nestedProperties: { role: { ref: 'OrganizationMemberRole' } },
-            validators: {},
-        },
-    },
-    // WARNING: This file was auto-generated with tsoa. Please do not modify it. Re-run tsoa to re-generate this file: https://github.com/lukeautry/tsoa
     OrganizationMemberProfileUpdate: {
         dataType: 'refAlias',
         type: {
-            ref: 'Partial_Pick_OrganizationMemberProfile.role__',
+            dataType: 'nestedObjectLiteral',
+            nestedProperties: {
+                role: { ref: 'OrganizationMemberRole', required: true },
+            },
```

Comment:
these changes are from this PR right? https://github.com/lightdash/lightdash/pull/5954

Reply:
yep make sure it's rebased on `main` ! Maybe something changed here

Reply:
Yes, these are the files "unlited", I rebased from master just now